pkg/instrumentation/metrics.go

```diff
@@ -56,28 +73,42 @@ func NewMetrics(reg *prometheus.Registry) *Metrics {
 			Name:      RepositoryConfigsTotal,
 			Help:      "Number of repository configurations",
 		}),
-		PublicRepositoriesNotIntrospectedLast24HoursTotal: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+		PublicRepositories36HourIntrospectionTotal: *promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
 			Namespace: NameSpace,
-			Name:      PublicRepositoriesNotIntrospectedLast24HoursTotal,
-			Help:      "Number of public repositories not introspected into the last 24 hours",
-		}),
+			Name:      PublicRepositories36HourIntrospectionTotal,
+			Help:      "Breakdown of public repository count by those that attempted introspection and those that were missed and were not.",
+		}, []string{"status"}),
 		PublicRepositoriesWithFailedIntrospectionTotal: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
 			Namespace: NameSpace,
 			Name:      PublicRepositoriesWithFailedIntrospectionTotal,
 			Help:      "Number of repositories with failed introspection",
 		}),
-		NonPublicRepositoriesNotIntrospectedLast24HoursTotal: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+		CustomRepositories36HourIntrospectionTotal: *promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
 			Namespace: NameSpace,
-			Name:      NonPublicRepositoriesNotIntrospectedLast24HoursTotal,
-			Help:      "Number of non public repositories not introspected in the last 24 hours",
-		}),
+			Name:      CustomRepositories36HourIntrospectionTotal,
+			Help:      "Breakdown of custom repository count by those that attempted introspection and those that were missed and were not.",
+		}, []string{"status"}),
 	}
 
 	reg.MustRegister(collectors.NewBuildInfoCollector())
 
 	return metrics
 }
 
+func (m *Metrics) RecordKafkaMessageStatus(success bool) {
```

Comment:
```suggestion
func (m *Metrics) RecordKafkaMessageResult(success bool) {
```
small thing, but I guess it would also make sense to rename this (here and elsewhere) since the name of the metric has changed